applications/portal/backend/api/controllers/search_controller.py

```diff
@@ -18,7 +18,7 @@ def fts_antibodies(page: int = 1, size: int = 50, search: str = '') -> Paginated
     return search_service.fts_antibodies(page, size, search)
 
 
-def filter_antibodies(filter_request: FilterRequest) -> PaginatedAntibodies:
-    if filter_request.size > 100:
+def filter_antibodies(body: FilterRequest) -> PaginatedAntibodies:
+    if body.size > 100:
         raise HTTPException(status_code=400, detail="Size must be less than 100")
-    return search_service.filter_antibodies(filter_request)
+    return search_service.filter_antibodies(body)
```

Comment:
Needed, because the openapi genapi.sh updates the main.py with body=body in the function.

<img width="336" alt="image" src="https://github.com/MetaCell/scicrunch-antibody-registry/assets/59233227/fa55181e-0437-4c8f-aa78-8555e4ec2c2b">